.github/actions/setup/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Setup
+description: "Check out repo and run yarn install with cache if found"
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
```

Comment:
Any way we can mention LTS here instead of hardcoding the value 16?

Reply:
we use `lts/*` in `.nvmrc` so it does make sense

Reply:
good point. i was following the circleci config

Reply:
Yeh

Reply:
Directly using .nvmrc now :D

```bash
Run actions/setup-node@v3
Node version file is not JSON file
Resolved .nvmrc as lts/*
Attempt to resolve LTS alias from manifest...
Attempt to resolve the latest version from manifest...
Resolved as '16.17.1'
Found in cache @ /opt/hostedtoolcache/node/16.17.1/x[64](https://github.com/OffchainLabs/arb-token-bridge/actions/runs/3225681980/jobs/5278282586#step:3:75)
```

https://github.com/OffchainLabs/arb-token-bridge/actions/runs/3225681980/jobs/5278282586